common/src/main/java/com/wynntils/services/destination/DestinationService.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.destination;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.components.Service;
+import com.wynntils.core.net.Download;
+import com.wynntils.core.net.UrlId;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DestinationService extends Service {
```

Comment:
This is not a `Service`, it is a `Model`. If you ever gets unsure, read the description in `Model.java` and `Service.java`:

```
 * Models are our representation of the Wynncraft world, game state and Wynncraft
 * metadata. In general, any non-trivial interesting aspect of Wynncraft that is
 * provided through vanilla Minecraft elements, should be parsed and provided by
 * a Model.
 ```
and
```
 * Services provide support to consumers for anything that is not specifically
 * related to the Wynncraft model, either since it is an external service
 * (3rd party library, or system/environment interaction), or since it is something
 * that we augment the Wynncraft world with, but the Wynncraft servers has no knowledge
 * about. In other words, this is a "misc" category, where everything consumers need
 * that is neither a manager, handler or manager can be placed.
```